tools/ci/setup_ruby_env.sh

```diff
@@ -1,3 +1,4 @@
 ./tools/ci/setup_ruby_environment.rb
+bundle config --local build.sassc --disable-march-tune-native
```

Comment:
What is going on here? Why is this needed?

Reply:
There is more context around why this was needed here ManageIQ/manageiq#19520
Essentially the build fails while installing (or building) this gem if the --disable-march-tune-native flag is not set for ppc64le